dashboards-observability/.cypress/utils/event_constants.js

```diff
@@ -70,4 +74,21 @@ export const landOnPanels = () => {
     `${Cypress.env('opensearchDashboards')}/app/observability-dashboards#/operational_panels`
   );
   cy.wait(delay);
+};
+
+export const renderTreeMapchart = () => {
```

Comment:
Take up a separate task to move all these utility functions into a util files instead of constants file. the filename shows that, this file is going to have definition of constants.